### PR TITLE
Update the YouTube guidelines

### PR DESCRIPTION
**Pull request by pymander**
This updates the YouTube guidelines with feedback from the community and formatting changes to make it match the Markdown style guidelines.

Co-authored-by: Aaron Epperson <aaron@aaronepperson.com>

-------

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: communication/youtube/youtube-guidelines.md [hunk 1]
```diff
@@ -56,191 +40,222 @@ Hours, will be shared with the appropriate point(s) of contact.
...
-permissions in your YouTube settings]. If you have any questions reach out to
-the [YouTube admins] or [SIG Contributor Experience].
+**NOTE:** If you're using a Google Workspace account (formerly known as G Suite)
+you may need to [update the permissions in your YouTube settings][]. If you have
+any questions, reach out to the [YouTube admins][] or
+[SIG Contributor Experience][]. You may need to reach out to someone at your
+organization if you do not have access to Google Workspace Admin permissions.
 
-**NOTE:** Both public and private steering meeting recordings should be made public.
+**NOTE:** Both public and private steering meeting recordings should be made
```
Comment (markyjackson-taulia) [inline communication/youtube/youtube-guidelines.md:49]: I don't think private meetings are public. @mrbobbytables is this correct?

File: communication/youtube/youtube-guidelines.md [hunk 2]
```diff
@@ -56,191 +40,222 @@ Hours, will be shared with the appropriate point(s) of contact.
...
+    should **only** be contacted if the issue cannot be resolved by one of the
+    playlist owners.
+
+-   Upload responsibilities belong to the group leaders or other appropriate
+    contacts. YouTube admins should **only** be contacted if the issue cannot be
+    resolved by the playlist owners.
+
+-   Please post only related content, for example meeting recordings, in the
+    appropriate playlists.
+    -   Posting of any exceedingly inappropriate content (i.e. NSFW content)
```
Comment (markyjackson-taulia) [inline communication/youtube/youtube-guidelines.md:66]: I think the word exceedingly should be removed. This sounds as though if it is not exceedingly, it is appropriate.

File: communication/youtube/youtube-guidelines.md [hunk 3]
```diff
@@ -56,191 +40,222 @@ Hours, will be shared with the appropriate point(s) of contact.
...
+    -   Save the changes and the order should automatically be updated.
+
+-   Please do not remove any already-published content from the playlists
+    without checking with the YouTube admins.
+
+-   For any small issues that arise, for example improper naming or ordering,
+    you may be asked by the YouTube admins to attempt to resolve the issue
+    yourself.
+
+-   Any egregious or habitual violations (3 or more per quarter) of the above
```
Comment (markyjackson-taulia) [inline communication/youtube/youtube-guidelines.md:86]: I think this should be discussed more. Audit trails do not exist in YouTube so enforcement without concrete proof is setting us up for a bad time

File: communication/youtube/youtube-guidelines.md [hunk 4]
```diff
@@ -56,191 +40,222 @@ Hours, will be shared with the appropriate point(s) of contact.
...
+improve the job of admins and help make things more streamlined. Please, reach
+out if you can help or have any ideas.
 
-Moderation responsibilities for YouTube admins is minimal and is centered around
-checking and removing any potential comments that would violate the [Code of
-Conduct][coc]. Any potential violations should sent to <conduct@kubernetes.io>.
+#### Splain.io
 
-### Trimming and Editing Recordings
+One feature used by several SIGs is splain.io. This tool creates a pipeline
```
Comment (markyjackson-taulia) [inline communication/youtube/youtube-guidelines.md:141]: We may need to remove this because splain is very buggy and more often than not does not work

**Review by markyjackson-taulia — CHANGES_REQUESTED** [review]

File: communication/youtube/youtube-guidelines.md (hunk 1 above)
Comment (mrbobbytables) [reply communication/youtube/youtube-guidelines.md:49]: Private recordings should be made public. 👍 steering only records the portions of the meeting that should be public. It seems a little weird, but thats the name that the meeting comes over with >_<

File: communication/youtube/youtube-guidelines.md (hunk 3 above)
Comment (mrbobbytables) [reply communication/youtube/youtube-guidelines.md:86]: Yeah, this isn't followed well and isn't enforced. There's too much overhead in it already. I do think we should tackle that in a separate issue/PR - this one is style guide changes.

File: communication/youtube/youtube-guidelines.md (hunk 4 above)
Comment (mrbobbytables) [reply communication/youtube/youtube-guidelines.md:141]: I'd leave as is till we get off splain =/ YT admins may still need the instructions to troubleshoot (I've had to re-setup splain for several groups to "restart" their pipeline)

**Review by mrbobbytables — COMMENTED** [review]
Awesome! Thanks @pymander :)
/approve
/hold
for further comments

**Review by mrbobbytables — COMMENTED** [review]
/lgtm
/hold cancel

**Pull request merged**